spec/requests/admin/users_controller_spec.rb

```diff
@@ -113,7 +131,7 @@
         Fabricate(:user, ip_address: "88.88.88.88")
         Fabricate(:admin, ip_address: user.ip_address)
         Fabricate(:moderator, ip_address: user.ip_address)
-        similar_user = Fabricate(:user, ip_address: user.ip_address)
+        _similar_user = Fabricate(:user, ip_address: user.ip_address)
```

Comment:
If this and [this](https://github.com/discourse/discourse/pull/30635/files#diff-93747edc7738309ae40af19243f62865c5eaf9b288823ff8aea117ef0cebf5faR2158) variables are not used, maybe we can skip assignment?

Reply:
I am mixed on this, in that the test author I guess wanted to convey meaning here, so we will lose that meaning or be forced to add another comment.